state_machine/frame/main.py

```diff
@@ -1,7 +1,8 @@
 import traceback
-from lib.pycubed import pocketqube as cubesat
+import lib.pycubed
 import state_machine
 
+cubesat = lib.pycubed.Satellite()

```

Comment:
Why did we make this change?

Reply:
I've commented reasoning / thoughts / potential solutions on the corresponding comment you made on pycubed.py.